docs/pages/guides/ios-developer-mode.mdx

```diff
@@ -7,54 +7,70 @@ description: Learn how to enable iOS developer mode on iOS 16 and above to run i
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 import { Step } from '~/ui/components/Step';
 
-On devices running iOS 16 and above, you will need to enable a special OS-level **Developer Mode** setting before you can run [internal distribution](/build/internal-distribution) builds (including those built with EAS) or local development builds installed on the device.
+> This does not apply to builds signed using enterprise provisioning, or to any builds installed on an iOS Simulator.
 
-To enable it, you do not need to connect the device to a computer or use Xcode.
+Devices running iOS 16 and above need to enable OS-level **Developer Mode** setting before they can run [internal distribution](/build/internal-distribution) builds (including those built with EAS) or local development builds after installing them on the device.
 
-> This does not apply to builds signed using enterprise provisioning, nor to any builds installed on an iOS Simulator.
+## Prerequisites
 
-Follow the instructions below (once per device).
+The instructions specified below need to be followed once per device. **You'll also need to install your development build on your device before you can enable the Developer Mode.**
+
+## Enable Developer Mode
 
 <Step label="1">
-Install the build on your device and press the app icon.
+
+After creating your first development build, follow the instructions from the Expo dashboard to install it on your iOS device.
```

Comment:
I often share a link to this section with individuals interested in testing through internal distribution, many of whom are not developers. I believe the phrase "After creating your first development build" might be confusing to them. Since this section primarily focuses on enabling Developer Mode on iOS, I suggest either removing or modifying that particular phrase.

Reply:
If they are testing internal distribution,  they'll have to install it on their device, and for that reason this section talks about what to do when they're installing the development build.

Maybe we can improve the phrasing here.

Reply:
Removed these instructions from step 1 and added them to prerequisites.